app/src/main/java/org/simple/clinic/patient/OngoingNewPatientEntry.kt

```diff
@@ -119,15 +119,19 @@ data class OngoingNewPatientEntry(
   fun withZone(zone: String): OngoingNewPatientEntry =
       copy(address = addressOrBlank().copy(zone = zone))
 
-  fun withPatientPrefillInfo(patientProfileInfo: PatientPrefillInfo, identifier: Identifier, dateTimeFormatter: DateTimeFormatter): OngoingNewPatientEntry =
+  fun withPatientPrefillInfo(
+      patientProfileInfo: PatientPrefillInfo,
+      identifier: Identifier,
+      dateTimeFormatter: DateTimeFormatter
+  ): OngoingNewPatientEntry =
       copy(
           personalDetails = PersonalDetails(
               fullName = patientProfileInfo.fullName,
               dateOfBirth = patientProfileInfo.dateOfBirth.format(dateTimeFormatter),
               gender = patientProfileInfo.gender,
               age = null),
           address = addressOrBlank().withColonyOrVillage(patientProfileInfo.address),
-          identifier = identifier)
+          alternativeId = identifier)
```

Comment:
Can we rename this `alternateId` to keep the naming convention consistent across for alternate id?

Reply:
`alternativeId` is the name given to the `OngoingNewPatientEntry` parameter. I'll change the places where I'm referring to it as `alternateId`. Sounds good?

Reply:
Actually in the `EditPatientScreen`, `PatientEntryScreen` and `PatientSummaryScreen` we're using `alternateId`. It would be inconsistent if we have `alternativeId` in `OngoingNewPatientEntry` for the additional identifier. 
@msasikanth What do you think about this?

Reply:
It's very inconsistent. Even in `PatientEntryScreen` it is being referred as `alternativeId` in some places. i'll make this change.

Reply:
TBH not that big of a deal. But feel free to make the change if you want. Ideally, it should be named Alternative, since this is a secondary identifier option.

Reply:
For now to maintain consistency renaming it to `alternateId`

Reply:
Altho everywhere both of these names are used. I see a mixture of `alternative` and `alternate` in all the screens